libs/platform/src/lib/components/page-footer/page-footer.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input, TemplateRef, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'fdp-page-footer',
+    templateUrl: './page-footer.component.html',
+    styleUrls: ['./page-footer.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+export class PlatformFooterComponent {
+    /** logo template accpets the generic html */
+    @Input()
+    logo: TemplateRef<any>;
+    /** content template accpets the generic html */
+    @Input()
+    content: TemplateRef<any>;
+    /** copyright template accpets the generic html */
+    @Input()
+    copyright: TemplateRef<any>;
```

Comment:
Wouldn't it be more natural to pass these options using `ng-content` slots?

Reply:
No I see this is by design https://app.gitbook.com/@fundamental-ngx/s/docs/styles-platform-footer-page-footer